src/components/gltf-model.js

```diff
@@ -0,0 +1,61 @@
+var registerComponent = require('../core/component').registerComponent;
+var THREE = require('../lib/three');
+
+var GLTFLoader = THREE.glTFLoader; // To appease the linter
+var loader = new GLTFLoader();
+var camera = null;
+var gltf = null;
+
+module.exports.Component = registerComponent('gltf-model', {
+  schema: {
+    type: 'src'
+  },
+
+  init: function () {
+    this.model = null;
+  },
+
+  update: function () {
+    var self = this;
+    var el = this.el;
+    var src = this.data;
+
+    camera = this.el.sceneEl.camera;
+    if (!src) { return; }
+
+    this.remove();
+
+    loader.load(src, function (gltfModel) {
+      var i = null;
+      var len = null;
+      var animation = null;
+
+      gltf = gltfModel;
+
+      self.model = gltfModel.scene;
+      el.setObject3D('mesh', self.model);
+      el.emit('model-loaded', {format: 'gltf', model: self.model});
+
+      if (gltf.animations && gltf.animations.length) {
+        len = gltf.animations.length;
+        for (i = 0; i < len; i++) {
+          animation = gltf.animations[i];
+          animation.loop = true;
+          animation.play();
+        }
+      }
+    });
+  },
+
+  remove: function () {
+    if (!this.model) { return; }
+    this.el.removeObject3D('mesh');
+  },
+
+  tick: function () {
+    if (gltf && camera) {
+      THREE.glTFAnimator.update();
```

Comment:
We're actually talking two things here:

1) Is THREE.AnimationMixer and glTFAnimator both needed?  We should probably understand this better before adding it in. 

2) Aside from THREE.AnimationMixer, should glTFAnimator be called for each object or only once for the app?  Even if it checks for duplicate calls, I think it would be clearer and technically more efficient to only call it once; but if we want to keep it to the glTF component, we might want to at least move it to a controller to avoid the duplicate calls.  My vote would just be put it in the event loop (wherever that is... still new here :)